apis/v1alpha1/shared_types.go

```diff
@@ -211,6 +211,19 @@ type RouteStatus struct {
 	//
 	// +kubebuilder:validation:MaxItems=100
 	Gateways []RouteGatewayStatus `json:"gateways"`
+
+	// AdmittedGateways is a list of Gateways that have admitted this route.
+	// This list includes the namespace and name of each Gateway in a
+	// namespace/name format. This field is primarily used to provide kubectl
+	// output.
+	//
+	// A maximum of 100 Gateways will be represented in this list. If this list
+	// is full, there may be additional Gateways using this Route that are not
+	// included in the list. An empty list means the route has not been admitted
+	// by any Gateway.
+	//
+	// +kubebuilder:validation:MaxItems=100
+	AdmittedGateways []string `json:"admittedGateways"`
```

Comment:
I'm not tied to this name. I also considered `AdmittingGateways`, `ConnectedGateways`, or `ActiveGateways` if any of those seem better. Also open to other names.

Reply:
I guess the question is what are we trying to convey? If the status of the object was `Admitted: false` because there's some sort of error, should it still show in this list? 

If that's the case then maybe `admittedGateways` isn't the right name (but was my first thought as well). =)

Possibly `selectedGateways` but that implies the opposite ordering (the route doesn't select the gateway).

Reply:
Yeah I think the most useful thing we can show here for kubectl is the Gateways that are actively using this Route. I was thinking that if a user expects a Gateway to be in the list and it isn't they can dig into the full yaml output/status.

Reply:
I agree that `Admitted` doesn't seem right, but I can't think of anything else that doesn't sound bad. `InUseByGateways` is probably accurate, but a bit of a mouthful.